common/management/commands/create_test_data.py

```diff
@@ -0,0 +1,1775 @@
+        )
+        max_hours = math.ceil(
+            reservation_unit.max_reservation_duration.total_seconds() / 3600
+        )
+
+        begin = past_date_start + timedelta(days=i)
+        begin = begin.replace(
+            hour=random.choice(range(8, 20 - max_hours)),
+            minute=random.choice([0, 15, 30, 45]),
+        )
+        end = begin + timedelta(hours=random.choice(range(min_hours, max_hours)))
+
+        state = STATE_CHOICES.CONFIRMED
+
+        pricing: ReservationUnitPricing = random.choice(
+            list(reservation_unit.pricings.all())
+        )
+        if pricing.highest_price != Decimal("0"):
+            state = random.choice(
+                [STATE_CHOICES.WAITING_FOR_PAYMENT, STATE_CHOICES.CONFIRMED]
+            )
+
+        deny_reason: ReservationDenyReason | None = None
+        cancel_reason = random.choices(
+            [None, random.choice(cancel_reasons)], weights=[3, 1]
+        )[0]
+        if cancel_reason is None:
+            deny_reason = random.choices(
+                [None, random.choice(deny_reasons)], weights=[3, 1]
+            )[0]
+            if deny_reason is not None:
+                state = STATE_CHOICES.DENIED
+        else:
+            state = STATE_CHOICES.CANCELLED
+
+        reservation = Reservation(
+            age_group=random.choice(age_groups),
+            applying_for_free_of_charge=random.choice([True, False]),
+            begin=begin,
+            billing_address_city=faker_fi.city(),
+            billing_address_street=faker_fi.street_name(),
+            billing_address_zip=faker_fi.postcode(),
+            billing_email=faker_fi.email(),
+            billing_first_name=user.first_name,
+            billing_last_name=user.last_name,
+            billing_phone=faker_fi.phone_number(),
+            buffer_time_after=timedelta(hours=random.choice(range(2))),
+            buffer_time_before=timedelta(hours=random.choice(range(2))),
+            cancel_reason=cancel_reason,
+            confirmed_at=begin,
+            deny_reason=deny_reason,
+            description=faker_fi.sentence(),
+            end=end,
+            free_of_charge_reason=faker_fi.sentence(),
+            name=f"Reservation {i}",
+            num_persons=persons,
+            price=pricing.highest_price,
+            price_net=pricing.highest_price_net,
+            purpose=random.choice(reservation_purposes),
+            reservee_address_city=faker_fi.city(),
+            reservee_address_street=faker_fi.street_name(),
+            reservee_address_zip=faker_fi.postcode(),
+            reservee_email=faker_fi.email(),
+            reservee_first_name=user.first_name,
+            reservee_id=faker_fi.company_business_id(),
+            reservee_language="fi",
+            reservee_last_name=user.last_name,
+            reservee_organisation_name=faker_fi.company(),
+            reservee_phone=faker_fi.phone_number(),
+            reservee_type=CUSTOMER_TYPES.CUSTOMER_TYPE_INDIVIDUAL,
```

Comment:
`reservee_type`:

- CUSTOMER_TYPE_INDIVIDUAL: no reservee/business_id or reservee_organisation_name
- CUSTOMER_TYPE_BUSINESS: has those
- CUSTOMER_TYPE_NONPROFIT: has both unless `reservee_is_unregistered_association` is true in which case only the org.name
(The latter two are OK to have the same person details are INDIVIDUAL; it is the contact person info then.)

Reply:
Reservee type and the related fields are now determined according to the above rules.